english/dream/lucid.md

```diff
@@ -0,0 +1,8 @@
+You’ve had some experience with lucid dreaming and begin to manipulate the dream so you have an escape from this
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.